src/ontology/edits/oeo-physical.omn

```diff
@@ -183,43 +171,10 @@ ObjectProperty: has_physical_input
     
 ObjectProperty: has_physical_output
 
-    Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "A relation that holds between a transformation and its output."@en
```

Comment:
Why did you delete this definition?

Reply:
it was doubled

Reply:
yes seems like it lost it's def now in oeo-physical I'll fix it

Reply:
I added it again in oeo-physical but now the def is doubled again in oeo because the property is defined in oeo-model too.

Reply:
Is this really correct now?

The old version was:
`<http://purl.obolibrary.org/obo/IAO_0000115> "A relation that holds between a transformation and its output."@en`

The new version is:
`definition "A relation that holds between a transformation and its output."`

Reply:
yes, it's not the same but equivalent.  

Detailed version: We have three types of "definition" annotations right now and not all are in all modules, so I tried to use the same one in oeo-model and oeo-physical in the hope to get oeo to realise they are identical. It didn't work and I didn't change the annotation type back because it's irrelevant.

Reply:
We should address that someday though

Reply:
The duplication of annotations if entities are defined in more than one module is an annoying "feature"/side effect of the way that the owl modularisation works. Ideally, all entities should have just one primary location where they are defined. We may want to consider having an oeo-relations module in which our "own" relations are defined, that is imported in all modules.

Reply:
I had the same idea of a relations module but had no idea whether it is possible.